docker/entrypoint.sh

```diff
@@ -1,7 +1,19 @@
 #!/bin/bash
 set -e
 
-# run aesmd in the background
-/opt/intel/sgx-aesm-service/aesm/aesm_service
+# Check if the first argument is "mrenclave"
+if [ "$1" = "mrenclave" ]; then
+    # If "mrenclave" is provided, execute the corresponding command
+    $SGX_ENCLAVE_SIGNER dump \
+      -enclave /usr/local/bin/enclave.signed.so \
+      -dumpfile df.out && \
+        /usr/local/bin/extract_identity < df.out && rm df.out | grep -oP ':\s*\K[a-fA-F0-9]+'
 
-exec /usr/local/bin/integritee-service "${@}"
+else
+    # If no specific command is provided, execute the default unnamed command
+
+    # run aesmd in the background
+    /opt/intel/sgx-aesm-service/aesm/aesm_service
+
+    exec /usr/local/bin/integritee-service "${@}"
+fi
```

Comment:
The docker image has now a new command `mrenclave`, which will print the mrenclave. Otherwise, it will forward all inputs to the integritee-service and simply run that binary.